packages/devtools_app/lib/src/shared/diagnostics/inspector_service.dart

```diff
@@ -964,7 +964,7 @@ class ObjectGroup extends InspectorObjectGroupBase {
   }) {
     return parseDiagnosticsNodeDaemon(
       invokeServiceMethodDaemonParams(
-        WidgetInspectorServiceExtensions.getRootWidgetTree.name,
+        WidgetInspectorServiceExtensions.getRootWidget.name,
```

Comment:
please remove this change to the inspector service

Reply:
done, it was unintentional